GPU/Common/TextureCacheCommon.cpp

```diff
@@ -2364,6 +2358,25 @@ bool TextureCacheCommon::PrepareBuildTexture(BuildTexturePlan &plan, TexCacheEnt
 		}
 	}
 
+	if (plan.isVideo || isPPGETexture) {
+		plan.replaced = &replacer_.FindNone();
```

Comment:
Texture inis have a setting to allow video, which may in some cases make sense such as the doors that open in Valkyrie Profile, etc.

-[Unknown]